### PR TITLE
Creation of the user interface for the game

### PR DESCRIPTION
**Pull request by JohannRodriguez**
# User interface

- Created table to be displayed.
- User can put their name.
- User can type their move to play.
- Not a functional game yet.

### CODE REVIEW

**Comment by pelzolga123** [comment]
## Hi Johann!  
### You did a great job so far :+1: 
But still, there are changes required before this milestone can be approved:
* [ ] In this step, not everything must work, the point is to make sure that you thought about everything listed below. Basic game flow should be prepared: ([screenshot hint](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone2/logic.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/)):
* Game should inform the player if the selected move is invalid.
* Game should display board after the player move.
* Game should inform the player if the selected move is a winning move.
* Game should inform the player if the selected move is a draw move.
* Game should repeat all actions for the next player's move.

* [ ] A loop is needed in this game, that has all repeated actions for each move (recommended is `while` but anything else will do) ([OdinProject hint](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone2/loop.png)) The loop should be terminated when one player wins or when it's a draw. It's advised to use a terminate variable for this loop. Example:
```ruby
game_on = true

while game_on
  # loop for each move
  if winner 
    game_on = false
  end
end
```
### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

File: bin/main [hunk 1]
```diff
@@ -1,2 +1,50 @@
...
+    p "valid input"
+  when "9"
+    p "valid input"
+  else
+    p "invalid input"
+  end
+end
+
+p "Now winner, it's a match" if turns == 0
+turns.times {player_move}
```
Comment (stratospherique) [inline bin/main:50]: * [ ] Using out-of-nowhere numbers to terminate a loop, for example `5.times do ... end` is a signal that something is wrong. We recommend the usage of a terminating variable (`while game_on`) as suggested by @pelzolga123 .
- **[SUGGESTION]** The structure of the game's loop should be as follows:
```ruby
game_on = true

# initial instructions for player

while game_on
  # loop for each move
  # get a valid user input
  # update the board
  # check if the move results to a winner or a draw or keep playing
  if winner # and/or draw (the exact condition in this milestone is not important)
    game_on = false
  end
  # switch turns if game not finished
end
```

**Comment by Anna-Myzukina** [comment]
# Hi team! :clap: 
### Good job setting up the project you are missing just one requirement  in Milestone 1 :eyes: 

*  In your project should be `bin` directory with `main.rb` file  from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/)
screenshot
*  `main.rb` is [executable file](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/):

* Also I want run this file to Test if this executable file works with next command: 
`bin/main.rb` or `ruby bin/main.rb` and it should displays anything.

* Your `main.rb` should include `puts/print` and `get.chomp` commands from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/) 
screenshot 


### You still need to add 

- [ ]  [REQUIRED] game asks player to select from `available moves`

- [ ]  [REQUIRED] game informs player if selected move is `a winning move`

- [ ]  [REQUIRED] game informs player if selected move is `a draw move`

*The final point of this milestone is to make sure that you thought about all the actions needed to play Tic Tac Toe from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/)


### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*
*Happy coding * <g-emoji class="g-emoji" alias="computer" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f4bb.png">computer</g-emoji>

**Comment by pelzolga123** [comment]
## Great work, congratulation :+1: Project Approved.
You did all changes required previously. Go ahead and merge to development.
_Goodluck in your next milestone_ *Happy coding*

**Pull request merged**